cmd/acr/purge_test.go

```diff
@@ -414,7 +416,7 @@ func TestPurgeManifests(t *testing.T) {
 		assert.Equal(nil, err, "Error should be nil")
 		mockClient.AssertExpectations(t)
 	})
-	// Thirteenth test, if a manifest should be deleted but the delete or write enabled attribute is set to false it should not be deleted
+	// Thirteenth test, if a manifest should be deleted but the delete enabled attribute is set to false it should not be deleted
```

Comment:
Nit. The tests do not need to be numbered. Would also suggest spacing them for readability. Could also blend them into a loop with configurable fields since each just seems a variation of the other but I wouldn't call that a major concern.

Reply:
removed the numbering and added spacings.